docs/configuration.md

```diff
@@ -124,7 +124,7 @@ Property
 **envoy-control.envoy.snapshot.routing.service-tags.enabled**                               | If set to true, service tags routing will be enabled                                                                                                                                      | false
 **envoy-control.envoy.snapshot.routing.service-tags.metadata-key**                          | What key to use in endpoint metadata to store its service tags                                                                                                                            | tag
 **envoy-control.envoy.snapshot.routing.service-tags.header**                                | What header to use in service tag rules                                                                                                                                                   | x-service-tag
-**envoy-control.envoy.snapshot.routing.service-tags.routing-excluded-tags**                 | List of tags, that cannot be used for routing. Tags matching these patterns will be not saved in endpoint metadata                                                                        | empty list
+**envoy-control.envoy.snapshot.routing.service-tags.routing-excluded-tags**                 | List of tags predicates that cannot be used for routing. This supports an exact matching (just "string" - EXACT matching) prefixes (PREFIX matching) and regexes (REGEX matching)         | empty list 
```

Comment:
List of *tag* predicates

Reply:
maybe I'd add `suffix` also so all bases would be covered (well, maybe except `contains` ;) ?